themes-default/slim/src/index.js

```diff
@@ -77,21 +79,23 @@ const UTIL = {
         }
 
         const { body } = document;
-        $('[asset]').each(function() {
-            const asset = $(this).attr('asset');
-            const series = $(this).attr('series');
-            const path = apiRoot + 'series/' + series + '/asset/' + asset + '?api_key=' + apiKey;
-            if (this.tagName.toLowerCase() === 'img') {
-                const defaultPath = $(this).attr('src');
-                if ($(this).attr('lazy') === 'on') {
-                    $(this).attr('data-original', path);
+        [...$('[asset]')].forEach(element => {
```

Comment:
Looks like that block (from here to line 100) can just be removed.
There aren't any elements with an `asset` attribute.

Reply:
I tried removing this last time and was told there are still places using this?

Reply:
Yes, I did, and IIRC at the time there were. Now I'm sure there aren't any